app/src/main/java/com/teamagam/gimelgimel/app/network/services/message_polling/MessageLocalBroadcaster.java

```diff
@@ -0,0 +1,33 @@
+package com.teamagam.gimelgimel.app.network.services.message_polling;
+
+import android.content.Context;
+
+import com.teamagam.gimelgimel.app.model.ViewsModels.Message;
+import com.teamagam.gimelgimel.app.model.ViewsModels.MessageBroadcastReceiver;
+import com.teamagam.gimelgimel.app.utils.NetworkUtil;
+
+/**
+ * Wraps Local-broadcasting functionality
+ */
+public class MessageLocalBroadcaster implements IMessageBroadcaster {
```

Comment:
That way I can test PolledMessageBroadcaster.
If I don;'t , there's no way for me to mock the Broadcast